WordOfTheDay.PerformanceCheck/WordOfTheDay.PerformanceCheck/Request.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+using System.Threading.Tasks;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Diagnostics;
+
+namespace WordOfTheDay.PerformanceCheck
+{
+    public class Request
```

Comment:
Rename it as "RequestManager" or "Requester". "Request" sounds to me like a single small object, but your class has lots of responsibilities